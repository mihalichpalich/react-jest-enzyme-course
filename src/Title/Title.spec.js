import React from "react";

import Title from "./Title";

describe("Title component", () => {
  it('should render Title component with props', () => {
    const component = shallow(<Title title="test title"/>);

    expect(component).toMatchSnapshot();
  });

  it('should render Title component with props', () => {
    const component = shallow(<Title/>);

    expect(component).toMatchSnapshot();
  });
});
