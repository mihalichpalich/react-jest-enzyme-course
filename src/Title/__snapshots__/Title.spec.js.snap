// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render Title component with props 1`] = `
<h1
  className="title"
>
  test title
</h1>
`;

exports[`Title component should render Title component with props 2`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
